products/sle/main.pm

```diff
@@ -796,6 +796,15 @@ elsif (get_var("FIPS_TS")) {
         load_applicationstests;
     }
 }
+elsif (get_var('SECURITY')) {
+    prepare_target();
+    if (get_var('BOOT_HDD_IMAGE')) {
+        loadtest "console/consoletest_setup";
+    }
```

Comment:
Because these four lines are repeated I suggest to move these up to the FIPS block in L767, e.g.

```perl
elsif (get_var('FIPS_TS') || get_var('SECURITY')) {
    prepare_target();
    if (get_var('BOOT_HDD_IMAGE')) {
        loadtest 'console/consoletest_setup';
    }
    if (...fips...) {
    }
    elsif (check_var('SECURITY', 'apparmor_status')) {
    ...
    }
```

Reply:
I have discussed with @mitiao and he has also suggested that. I have a little concern about in future SECURITY cases will be different with FIPS. I will consider again and make change in next PR if necessary.

Reply:
Done!